cabal-install/cabal-install.cabal

```diff
@@ -102,6 +102,10 @@ source-repository head
   location: https://github.com/haskell/cabal/
   subdir:   cabal-install
 
+Flag old-bytestring
+  description:  Use bytestring == 0.9.* and bytestring-builder
```

Comment:
I'm afraid the description is slightly inaccurate regarding the bytestring version range